trust-store-trust-policy-specification.md

```diff
@@ -97,27 +97,46 @@ The trust policy is represented as JSON data structure as shown below:
 ```json
 {
     "version": "1.0",
-    "trustPolicies": {
-        "name": "trust-policy-name",
-        "trustStores": [ "trust-store-name-1", "trust-store-name-2" ],
-        "expiryValidations": {
-            "signatureExpiry": "enforce/warn",
-            "signingIdentityExpiry": "enforce/warn",
-            "timestampExpiry": "enforce/warn"
+    "trustPolicies": [
+        {
+            "name": "trust-policy-name-1",
+            "Scopes": ["wabbit-networks.io/software"],
+            "trustStores": [ "trust-store-name-1", "trust-store-name-2" ],
+            "expiryValidations": {
+                "signatureExpiry": "enforce/warn",
+                "signingIdentityExpiry": "enforce/warn",
+                "timestampExpiry": "enforce/warn"
+            },
+            "revocationValidations": {
+                "signingIdentityRevocation": "enforceWithFailOpen/enforceWithFailClose/warn/skip",
+                "timestampRevocation": "enforceWithFailOpen/enforceWithFailClose/warn/skip"
+            }
         },
-        "revocationValidations": {
-            "signingIdentityRevocation": "enforceWithFailOpen/enforceWithFailClose/warn/skip",
-            "timestampRevocation": "enforceWithFailOpen/enforceWithFailClose/warn/skip"
+        {
+            "name": "trust-policy-name-2",
+            "trustStores": [ "trust-store-name-1", "trust-store-name-2" ],
+            "expiryValidations": {
+                "signatureExpiry": "enforce/warn",
+                "signingIdentityExpiry": "enforce/warn",
+                "timestampExpiry": "enforce/warn"
+            },
+            "revocationValidations": {
+                "signingIdentityRevocation": "enforceWithFailOpen/enforceWithFailClose/warn/skip",
+                "timestampRevocation": "enforceWithFailOpen/enforceWithFailClose/warn/skip"
+            }
         }
-    }
+
+    ]
 }
 ```
 
 Property descriptions
 
 - **`version`**(*string*): This REQUIRED property is the version of the trust policy. The supported value is `1.0`.
-- **`trustPolicies`**(*string-object map*): This REQUIRED property represents a collection of trust policies. As of now Notary v2 only supports one trust policy.
-  - **`name`**(*string*): The name of the trust policy
+- **`trustPolicies`**(*string-array of objects map*): This REQUIRED property represents a collection of trust policies.
+  - **`name`**(*string*): The name of the trust policy.
+  - **`scope`**(*string*): The scope determines which trust policy is applicable for a given artifact. The scope field supports prefix-based filtering on registry-name/namespace+repository-name. For an artifact, if there is no applicable trust policy, then signature evaluation must be skipped.
```

Comment:
Should we allow wildcards in the scope? Some may want to trust `*.example.com/teamx/*` to trust the same key across a set of registry servers without naming each one separately.